decidim-accountability/app/commands/decidim/accountability/admin/update_imported_result.rb

```diff
@@ -5,84 +5,30 @@ module Accountability
     module Admin
       # This command is executed when the user changes a Result from the admin
       # panel.
-      class UpdateImportedResult < Decidim::Command
+      class UpdateImportedResult < Decidim::Commands::UpdateResource
+        include Decidim::Accountability::Admin::HasResultCommand
+
+        fetch_form_attributes :scope, :category, :title, :description, :start_date, :end_date, :progress,
+                              :decidim_accountability_status_id, :external_id, :weight
+
         # Initializes an UpdateResult Command.
```

Comment:
```suggestion
        # Initializes an UpdateImportedResult Command.
```

Reply:
done in 94771d78819c3002354367f20f75b28d814444b9